player/clipboard/clipboard-wayland.c

```diff
@@ -27,6 +27,9 @@
 #include "osdep/poll_wrapper.h"
 #include "osdep/threads.h"
 
+static const char MESSAGE_DEATH = 0;
+static const char MESSAGE_CREATE_SOURCES = 1;
```

Comment:
```suggestion
static const uint8_t MESSAGE_DEATH = 0;
static const uint8_t MESSAGE_CREATE_SOURCES = 1;
```

can also add `msg_t` or similar typedef for cleaner sizeof use.